tests/shared/src/test/scala/scala/meta/tests/parsers/BasePositionSuite.scala

```diff
@@ -0,0 +1,65 @@
+package scala.meta.tests.parsers
+
+import munit.FunSuite
+import scala.meta.parsers.Parse
+import scala.meta.internal.inputs._
+import munit.TestOptions
+import scala.meta.internal.trees.Origin
+import munit.Location
+import scala.meta.tests.parsers.MoreHelpers
+
+abstract class BasePositionSuite extends ParseSuite {
+  import scala.meta._
+  def defaultDialect: Dialect = dialects.Scala213
```

Comment:
```suggestion
  def defaultDialect: Dialect
```
I think it would be good to have it explicitely set in the implementing classes, so that it's clearer which dialect is each testing

Reply:
Good point. I made the dialect a constructor argument.